test/app/components/with_translations.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class WithTranslations < ActionView::Component::Base
```

Comment:
Might you be able to rename this to `TranslationsComponent`, to fit convention?

Reply:
Done ✔️